recipes/alembic/all/conanfile.py

```diff
@@ -44,9 +44,9 @@ def layout(self):
         cmake_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("imath/3.1.9", transitive_headers=True)
+        self.requires("imath/3.1.12", transitive_headers=True)
         if self.options.with_hdf5:
-            self.requires("hdf5/1.14.3")
+            self.requires("hdf5/1.14.6")
```

Comment:
```suggestion
            self.requires("hdf5/1.14.3")
```

This is very likely to cause conflicts, as 1.14.3 is reference everywhere else
is there a reason for these version bumps? Otherwise we will update it once we do it across the entire repo

Reply:
There is no reason, can I use this PR to use version range?

Reply:
@EstebanDugueperoux2 Unfortunately, not in this case. While the hdf5 has [explicit backward compatilibity](https://github.com/HDFGroup/hdf5/blob/develop/CONTRIBUTING.md?plain=1#L59) and fits for version ranges, it will result in the same rule as before: Conan will pick the latest version available and build with that, resulting in the same problem. So please, revert it and follow Luis's code suggestion.